### DISCUSSION TITLE
How to add Ignore Blank option in dataValidation

### DISCUSSION THREAD

**Discussion by rajeshpatel1993** [exceljs/exceljs#2473] [Q&A]
I want to add Ignore blank option for data validation , How can i do that? please help
Ex.
`worksheet.getCell('A1').dataValidation = {
  type: 'list',
  allowBlank: true,
  formulae: ['$D$5:$F$5']
};`

Now if that range contains empty , blank, null values it should ignored in dropdown?

**Comment by 1712853** [comment]
I face the same issue, can I answer how you fix it ?
Thank you so much!
